### PR TITLE
DOC: Start v1.1.4 release notes

### PR DESCRIPTION
**Pull request by dsaxton**
I think this requires the v1.1.3 tag in order to pass

### CODE REVIEW

**Comment by simonjayhawkins** [comment]
lgtm once v1.1.3 released.

**Comment by jorisvandenbossche** [comment]
This can already be merged in master I think? (only the backport should only be merged after 1.1.3)

**Comment by simonjayhawkins** [comment]
don't merge. if merged web and docs will fail until tag available.

File: doc/source/whatsnew/v1.1.3.rst [hunk 1]
```diff
@@ -73,4 +73,4 @@ Other
 Contributors
 ~~~~~~~~~~~~
 
-.. contributors:: v1.1.2..v1.1.3|HEAD
```
Comment (jorisvandenbossche) [inline doc/source/whatsnew/v1.1.3.rst:76]: Can you also remove this removal (and leave this for the actual release commits which also update the date etc), then it should pass?

Comment (dsaxton) [reply doc/source/whatsnew/v1.1.3.rst:76]: There's also a reference to v1.1.3 down below, should that also be removed, or do we wait for the tag?

**Comment by simonjayhawkins** [comment]
Thanks @dsaxton

**Pull request merged**